README.md

```diff
@@ -43,6 +43,18 @@ yarn dev
 
 ## Build the Distributable
 
+> [!WARNING]
+> The default bootsrap, signaling and ICE servers (used for connection establishment among peers)
+> in kangaroo.config.ts have no availability guarantees whatsoever and are meant for testing
+> purposes only.
+>
+> If you want to deploy your app to end-users, make sure to run your own
+> instances of these servers or use servers that have guaranteed availability for the lifetime
+> of your apps network(s).
+>
+> **Changing these URLs *after* deployment of your app can result in a newtork partition**.
```

Comment:
```suggestion
> **Two peers must share the same signal and bootstrap servers to communicate, and thus changing these URLs *after* deployment of your app can result in a newtork partition**.
```

Is that clearer?

Reply:
I think strictly speaking, the ICE one's don't need to match necessarily if I'm not mistaken. But the bootstrap and signaling ones do.